renku/cli/init.py

```diff
@@ -443,6 +446,14 @@ def init(
         except GitCommandError as e:
             click.UsageError(e)
 
+    # supply additional parameters
+    parameter["__template_source__"] = template_source
```

Comment:
should this be renamed to `metadata` for consistency? Just a suggestion, feel free to ignore.